site/website/blog/2021-02-02-jet-44-is-released.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Jet 4.4 Released
+author: Marko Topolnik
+authorURL: https://twitter.com/mtopolnik
+authorImageURL: https://pbs.twimg.com/profile_images/922726943614783488/Pb5DDGWF_400x400.jpg
+---
+
+Today we're releasing Hazelcast Jet 4.4 and we have some exciting new
+features!
+
+## Jet SQL
+
+Hazelcast Jet 4.4 brings you the first beta version of our SQL
+interface. You can now log into Jet from the command line and issue
+queries against the data sources you specify. They can be both data at
+rest (_batch_ sources) and live feeds (_streaming_ sources).
+
+If you have Docker at hand, here's something you can try out right now!
+(For examples that don't require Docker, go to the
+[docs](/docs/sql/intro).)
+
+```text
```

Comment:
bash?

Reply:
That was my first choice of course, but it renders very bad.